iconservice/deploy/icon_score_deploy_engine.py

```diff
@@ -213,7 +214,7 @@ def _on_deploy_for_builtin(self,
             pass
 
         try:
-            score = context.icon_score_mapper.load_score(score_address, next_tx_hash)
+            score = IconScoreContextUtil.icon_score_mapper.load_score(score_address, next_tx_hash)
```

Comment:
```suggestion
            score = IconScoreContextUtil.load_score(context, score_address, next_tx_hash)
```
Preserving icon_score_mapper in IconScoreContext